### DISCUSSION TITLE
GH Actions builds

### DISCUSSION THREAD

**Discussion by jhollowe** [bmeares/Meerschaum#49] [Ideas]
I see that there is a not-used GH Action for building and pushing images to docker registries. Would a PR to get that working be appreciated?

**Comment by bmeares** [comment]
Hey John, the GH Workflow was an experiment to replicate some of my [build scripts](https://github.com/bmeares/Meerschaum/tree/master/scripts/docker). However, `docker buildx` kept failing and was difficult to manage with GH Actions, so I'm still building with my bash scripts.

If you can get it working with my `scripts/docker/buildx.sh` script, a PR would be appreciated! However, `docker buildx` is pretty intensive and slow to build (building multiple tags for several architectures like x86 and ARMv7) and a bit difficult to debug, so I'm not sure if GH Actions is the best platform for building. But I would love to integrate it and learn some best practices for continuous deployment!

Lastly, if you're planning on using Meerschaum soon, try the `0.3.x` branch which includes some important bugfixes over the `0.2.x` release in `master`. I'll be merging `0.3` into `master` after finals are finished.
